py/desispec/scripts/proc_night.py

```diff
@@ -53,7 +53,8 @@ def proc_night(night=None, proc_obstypes=None, z_submit_types=None,
                path_to_data=None, exp_obstypes=None, camword=None,
                badcamword=None, badamps=None, exps_to_ignore=None,
                sub_wait_time=0.5, verbose=False, dont_require_cals=False,
-               psf_linking_without_fflat=False):
+               psf_linking_without_fflat=False,
+               still_acquiring=None):
```

Comment:
Change to False

Reply:
Done; see commentary on desi_proc_night comment.